### PR TITLE
feat desktop notification system

### PR DESCRIPTION
**Pull request by zjy365**

### CODE REVIEW

**Comment by zjy365** [comment]
image
image

**Comment by zjy365** [comment]
image

File: frontend/desktop/src/components/notification/index.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+export default function Notification(props: TNotification) {
+  const { isShow, onClose, onAmount } = props;
+  const [activeTab, setActiveTab] = useState<'read' | 'unread'>('unread');
+  const [activePage, setActivePage] = useState<'index' | 'detail'>('index');
+  const [msgDetail, setMsgDetail] = useState<NotificationItem>();
+
+  const { data, refetch } = useQuery(
+    ['getAwsAll'],
+    async () => await request('/api/notification/list')
```
Comment (c121914yu) [inline frontend/desktop/src/components/notification/index.tsx:42]: () => request('/api/notification/list')

File: frontend/desktop/src/components/notification/index.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  const unread_notes = useMemo(() => {
+    return data?.data?.items?.filter((item: NotificationItem) => !item?.metadata?.labels?.isRead);
+  }, [data?.data?.items]);
+
+  const read_notes = useMemo(() => {
+    return data?.data?.items?.filter((item: NotificationItem) => item?.metadata?.labels?.isRead);
+  }, [data?.data?.items]);
+
+  useEffect(() => {
+    onAmount(unread_notes?.length || 0);
```
Comment (c121914yu) [inline frontend/desktop/src/components/notification/index.tsx:54]: Use useQuery uniformly

**Review by c121914yu — APPROVED** [review]

**Review by c121914yu — APPROVED** [review]

**Review by c121914yu — APPROVED** [review]

**Review by zzjin — APPROVED** [review]

**Pull request merged**